packages/turbo-ignore/utils.js

```diff
@@ -0,0 +1,69 @@
+const fs = require("fs");
+const path = require("path");
+const { findRootSync } = require("@manypkg/find-root");
+
+export function searchUp(pathName, cwd) {
+  const root = path.parse(cwd).root;
+
+  let found = false;
+
+  while (!found && cwd !== root) {
+    if (fs.existsSync(path.join(cwd, pathName))) {
+      found = true;
+      break;
+    }
+
+    cwd = path.dirname(cwd);
+  }
+
+  if (found) {
+    return cwd;
+  }
+
+  return null;
+}
+
+export function getScope() {
```

Comment:
My pitch is logic that looks like this:

`getScopeFromArgs(process.argv.slice()) || getScopeFromPath(process.cwd())`

(but tweaked since it would change how logging happens)